oc_logging/test/test.py

```
@@ -21,6 +21,16 @@ def _raise_zero_division():
         log.error("failed", exc_info=True, key="value")
 
 
+def _log_exception_object():
+    """Log an exception object as the event, the way log.exception(err) is often used"""
+    log = structlog.get_logger()
+
+    try:
+        raise ValueError("something went wrong")
+    except ValueError as err:
+        log.exception(err)
```

Ruff (0.16.1)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)